validations/src/validations.rs

```diff
@@ -943,6 +948,24 @@ fn create_expected_tally_transaction(
     Ok((ta_tx, dr_state.clone()))
 }
 
+pub fn calculate_errors_and_liars_count_from_tally(
+    tally_tx: &TallyTransaction,
+    dr_state: &DataRequestState,
+) -> (usize, usize) {
+    tally_tx
+        .out_of_consensus
+        .iter()
+        .fold((0, 0), |(l_count, e_count), x| {
+            if !dr_state.info.reveals.contains_key(x) {
```

Comment:
What is this branch? If an identity is out of consensus and is not a revealer? Shouldn't that be a lie?

Reply:
the function calculate_witness_rewards uses liars_count as a node that commit and reveal a value and it is a lie, so in that case we do not take into account the non-reveal that are calculated using another values... but yes, it could be misunderstanded